src/components/Tabs/Tabs.test.tsx

```diff
@@ -54,4 +54,120 @@ describe('<Tabs />', () => {
     });
   });
 
+  describe('Tab keyboard interactions', () => {
+    let mock;
+
+    beforeEach(() => {
+      mock = mount(
```

Comment:
While I'm not sure of the best term for this, I don't think `mock` is accurate and is potentially misleading. We use `cut` in other tests, which I honestly don't understand, but we should probably use the same name everywhere.